packages/ui/lib/components/card-info/CardInfo.tsx

```diff
@@ -0,0 +1,143 @@
+import { Divider, List, ListItem, ListItemText, Stack } from "@mui/material";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import React, { ReactNode } from "react";
+
+export interface CardInfoProps {
+  /** Generic bottom content _(will be displayed after `items`)_ */
+  bottomContent?: ReactNode;
+  /** Main card icon */
+  icon?: ReactNode;
+  /** Structured list of `CardInfoItem` */
+  items?: CardInfoItem[];
+  /** Main card title */
+  title?: string;
+  /** Generic top content _(will be displayed before `items`)_ */
+  topContent?: ReactNode;
+}
+
+export interface CardInfoItemsProps {
+  items: CardInfoItem[];
+}
+
+export interface CardInfoItem {
+  /** footer text _(will be displayed on bottom)_ */
+  footerText?: string;
+  /** Item label */
+  label: string;
+  /** Item value */
+  value: ReactNode | string;
+}
+
+const DATA_TEST_ID_PREFIX = "io-ipatente-card-info";
+
+const CARD_BACKGROUND_COLOR = "#F5F7FB";
```

Comment:
nit: using colors from theme settings instead of hardcoded values provides a single source of truth. This makes it easier to manage and update the entire theme consistently, allowing changes to be applied uniformly across the application.

Reply:
Yes you're right and we will review colors and styles globally as soon as we have an overview of everything (or almost everything) we will use.
For now it's premature, we are planning to do it in a few days after we finish the development of the shared components.

Reply:
@gunzip this is a first refactor 1110291.
We will refactor other global styles and colors in a few days after we finish the development of the shared components.

Reply:
@gunzip final refactor for global theme colors: 94bec71